### PR TITLE
HomeTask2

### PR DESCRIPTION
**Pull request by BayMaxMB**
I need suggestion on how to include routes/auth into server.js in a best way, I needed to include passport lib inside this auth.js fie

### CODE REVIEW

**Comment by BayMaxMB** [comment]
And please suggest me better option to generate unique integer number user ID

How to be 100% sure that ID generator cannot generate same ID after some time? Checking the whole database before assigning?

File: database.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
+const bcrypt = require('bcrypt');
+
+const users = [{
```
Comment (Lokki17) [inline database.js]: Move data to another file

File: routes/auth.js [hunk 2]
```diff
@@ -1,43 +1,63 @@
...
-		},
-		role: "user"
+const options = {
+	jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken(),
+	secretOrKey: process.env.JWT_SECRET,
+}
+passport.use(new JwtStrategy(options, jwtCallback));
+const auth = passport.authenticate('jwt', { session: false });
+
+router.post('/register', (req, res) => {
```
Comment (Lokki17) [inline routes/auth.js]: Move router hanlders (functions) to separate file

**Review by Lokki17 — CHANGES_REQUESTED** [review]
Fix remarks and merge

**Comment by Lokki17** [comment]
> And please suggest me better option to generate unique integer number user ID
> 
> How to be 100% sure that ID generator cannot generate same ID after some time? Checking the whole database before assigning?

https://stackoverflow.com/questions/23327010/how-to-generate-unique-id-with-node-js

**Comment by Lokki17** [comment]
> I need suggestion on how to include routes/auth into server.js in a best way, I needed to include passport lib inside this auth.js fie

I didn't get you

**Pull request merged**